src/sentry/snuba/discover.py

```diff
@@ -961,12 +963,16 @@ def histogram_query(
     columns = [] if key_column is None else [key_column]
     limit = len(fields) * num_buckets
 
+    orderby = [histogram_alias]
+    if order_by:
+        orderby.insert(0, order_by)
```

Comment:
Might not be doing what you expect, orderby will always be truthy cause of L966, and then inserting orderby into itself (TIL you can do that :eyes:)
```suggestion
    orderby = orderby + [histogram_alias]
```

Reply:
lol